.github/dependabot.yml

```
@@ -0,0 +1,13 @@
+version: 2
+updates:
+- package-ecosystem: maven
+  directory: "/"
+  schedule:
+    interval: weekly
```

Weekly dependency checks are more frequent than needed for plugins that rarely change.  The [tutoiral](https://www.jenkins.io/doc/developer/tutorial-improve/automate-dependency-update-checks/) recommends monthly as does the ["automating plugin releases"](https://www.jenkins.io/doc/developer/publishing/releasing-cd/#configure-dependabot-optional) page.
```suggestion
    interval: monthly
```